src/gst-plugins-ugly.mk

```diff
@@ -25,4 +25,8 @@ define $(PKG)_BUILD
         --docdir='$(1)/sink' \
         --with-html-dir='$(1)/sink'
     $(MAKE) -C '$(1)' -j '$(JOBS)' install
+
+    # some .dlls are installed to lib - no obvious way to change
+    $(and $(BUILD_SHARED),
```

Comment:
> gstreamer.mk doesn't use a indent for it.

I think, we can keep the current version, but it should be fixed someday. Maybe in another mass-cleanup effort like https://github.com/mxe/mxe/pull/1621

> Do you insist on change "and->if"?

I don't insist, but I am curious why `and` is used. It is inconsistent with other code. Is there a rationale of using it rather than `if`?